wandb/sdk/launch/sweeps/utils.py

```diff
@@ -227,9 +227,11 @@ def create_sweep_command_args(command: Dict) -> Dict[str, Any]:
     # (5) flags without equals (e.g. --foo bar)
     args_no_equals: List[str] = []
     for param, config in command["args"].items():
-        _value: Any = config.get("value", None)
-        if _value is None:
+        try:
```

Comment:
I'd add a comment here indicating why we're doing this.

Reply:
comments are a crutch, but you are the boss ;)